source/views/stoprint/components/error.js

```diff
@@ -49,6 +49,32 @@ export class StoPrintErrorView extends React.PureComponent<Props> {
 				)}
 			/>
 		)
+
+		const errorWithoutTimer = (
+			<ScrollView
+				contentContainerStyle={styles.content}
+				showsVerticalScrollIndicator={false}
+				style={styles.container}
+			>
+				<Icon color={sto.black} name={iconName} size={100} />
+				<NoticeView
```

Comment:
Isn't this the view that's provided for an invalid username/password? 

If so, we shouldn't send them to the Connection Issue page.

Reply:
It appears the connection issue page is being used as the error page for invalid username/password once we've confirmed credentials aren't blank. I agree with you –– we should render something else.

Reply:
If we route to a different view, is this approach valid?